spec/lib/bank_account/account_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+require "./lib/bank_account/account"
+
+describe BankAccount::Account do
+
+  context "#self.new" do
+    it "responds to class method called new" do
+      expect(BankAccount::Account).to respond_to :new
+    end
+
+    let (:neg_account) {BankAccount::Account.new(20, -100)}
+
+    it '#self.new with a negative balance raises an ArgumentError' do
+      expect{neg_account}.to raise_error(ArgumentError, "Invalid amount.")
+    end
+  end # context
+
+  let(:account) {BankAccount::Account.new(20, 100)}
```

Comment:
I almost missed this line when I was reading the spec. Generally, `let`s should be right at the top of the relevant `describe`/`context` block.